features/guidance/turn.feature

```diff
@@ -940,12 +940,12 @@ Feature: Simple Turns
 
         And the ways
             | nodes | name          | highway   | oneway |
-            | ab    | Molkenmarkt   | secondary | yes    |
+            | ab    | Stralauer Str | secondary | yes    |
```

Comment:
`ab` models this way https://www.openstreetmap.org/way/23812656 and is not Stralauer Str but Molkenmarkt instead.